kobuki_node/src/odometry.cpp

```diff
@@ -14,12 +14,12 @@
 #include <memory>
 #include <string>
 
-#include <geometry_msgs/msg/quaternion.hpp>
 #include <geometry_msgs/msg/transform_stamped.hpp>
 #include <nav_msgs/msg/odometry.hpp>
+#include <rclcpp/rclcpp.hpp>
 #include <tf2/LinearMath/Quaternion.h>
 
-#include <ecl/geometry.hpp>
+#include <ecl/geometry/legacy_pose2d.hpp>
```

Comment:
Need to deprecate this. Don't need to do it here though.

https://github.com/kobuki-base/kobuki_ros/issues/11

Reply:
I'll take a look.